docs/changelog.rst

```diff
@@ -6,6 +6,7 @@ A log of changes by version and date.
 ======= ========== =====
 Version Date       Notes
 ======= ========== =====
+1.30.0  12/15/2022 Added FlashArray 2.20 client
```

Comment:
nit: fixup the date when we are ready to release it

Reply:
👍